hadoop-ozone/client/src/test/java/org/apache/hadoop/ozone/client/TestOzoneECClient.java

```diff
@@ -573,8 +573,8 @@ public void testStripeWriteRetriesOn2Failures() throws IOException {
         nodesIndexesToMarkFailure);
```

Comment:
I find all of these changes are unrelated as well. I would suggest to repurpose other JIRA as cleanups task and do this changes. Hope that make sense to you.